tools/mysql/docker-init/initdb.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Exit upon any errors
+set -e
+
+cd /var/teletraan/tools
+
+./deploydb.sh -f ../sql/deploy.sql
+./upgrade.sh
```

Comment:
This script will be invoked by mysql container to create the database and update tables.